config/tools-manual.json

```diff
@@ -149,6 +149,18 @@
                     "technology": [".NET"],
                     "categories": ["code-first"]
                 }
+            },
+            {
+              "title": "nestjs-asyncapi",
+              "description": "Utilize decorators to generate AsyncAPI document utilizing DTOs (similar to @nestjs/swagger) and a web UI.",
+              "links": {
+                "repoUrl": "https://github.com/flamewow/nestjs-asyncapi"
+              },
+              "filters": {
+                "language": ["Typescript"],
```

Comment:
```suggestion
                "language": "Typescript",
```

The `language` field should be of string, not an array.

Reply:
> Hey @flamewow, thanks for adding a new tool to our website. But I would suggest you make `.asyncapi-tool` file in your tool repo and maintain the metadata you want to show for your tool in the website. In this way, you don't have to make PR to this repo for making changes related to the information of the tool.

Thanks, gonna do it that way

Reply:
@akshatnema added the file https://github.com/flamewow/nestjs-asyncapi/blob/main/.asyncapi-tool

wondering how sync files are discovered by the asyncapi/website, do I need to register the repository in somewhere?